KnowWhereGraph/kwg-faceted-search
# Table links

This PR completes the tables (some data was previously missing) and makes table elements HTML links. The table styling is far from ideal after this PR because of the new data additions. This will be addressed in the future.

To Test
1. Check this branch out and run
2. Navigate to each tab and make sure that every table element is a link, and that the links are correct.
